src/main/java/uk/gov/companieshouse/api/accounts/model/entity/smallfull/notes/relatedpartytransactions/RptTransactionEntity.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.api.accounts.model.entity.smallfull.notes.relatedpartytransactions;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.api.accounts.model.entity.BaseEntity;
+
+public class RptTransactionEntity extends BaseEntity {
```

Comment:
I believe we need `@Document` annotation here with the collection name notes otherwise it will create another collection with the class name

Reply:
I think it would go inside the notes collection as this is a note (`RptTransaction`) and not a `Transaction`